content/working_with/console/widgets/topology.md

```diff
@@ -19,12 +19,66 @@ The blueprint or deployment ID must be selected in one of the following ways:
 ### Presentation
 
 Each of the blueprint's nodes is displayed as a square container that can contain other nodes. 
-Each node has a name, and an icon to indicate its [node type]({{< relref "developer/blueprints/spec-node-types.md" >}}).
+Each node has a name, and an icon (upper right corner) to indicate its [node type]({{< relref "developer/blueprints/spec-node-types.md" >}}). 
 
 [Relationships]({{< relref "developer/blueprints/spec-relationships.md" >}}) between nodes are indicated with arrows that start at the connected node and end at the target node.
 
 The number of node instances is marked in a bullet beside the node's type icon.
-   
+
+Each node is provided with [plugin]({{< relref "developer/blueprints/spec-plugins.md" >}}) icon (top left corner) which displays the plugin from which the node comes from.
```

Comment:
Not every node type comes from a plugin - maybe worth to mention the generic "plug" icon that is used for built-in types?

Reply:
Yep, added more detailed description.